content/shmem_init_thread.tex

```diff
@@ -0,0 +1,55 @@
+\apisummary{
+Initializes the OpenSHMEM library, similar to \FUNC{shmem\_init}, and performs any
+initialization required for supporting the provided thread level.}
+
+%Initializes the \openshmem{} library, similar to \FUNC{shmem\_init}, and in addition performs the initialization required for thread-safe invocation of \openshmem{} functions.}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int shmem_init_thread(int requested, int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{requested}  {The thread level support requested by the user.
+}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+\FUNC{shmem\_init\_thread}  initializes the \openshmem{} library similar to
+\FUNC{shmem\_init}, and in addition performs the initialization required for
```

Comment:
Similar is an adjective and similarly is an adverb, so according to the rules of grammar, similarly is more correcter in the sentence above.  The awkwardness is real, though.  How about rephrasing to: "shmem_init_thread is similar to shmem_init; both initialize the OpenSHMEM library.  In addition, shmem_init_thread also allows the user to request a particular level of threading support."